### DISCUSSION TITLE
Leveling not Active after reboot Printer?

### DISCUSSION THREAD

**Discussion by tome9111991** [Jyers/Marlin#782] [Q&A] [answered]
Hello.

I use latest build of jyers (self compiled bltouch 5x5) 
In Level menu i hit "Leveling Active" the checkbox is checked.
I save settings and turn off/ turn on printer
And "Leveling Active" is uncheked?

Is my mesh still ative in a Print?
I use M420 S1 in startcode

**Comment by Tuncay-Ayhan** [comment]
> Hello.
> 
> I use latest build of jyers (self compiled bltouch 5x5)
> In Level menu i hit "Leveling Active" the checkbox is checked.
> I save settings and turn off/ turn on printer
> And "Leveling Active" is uncheked?
> 
> Is my mesh still ative in a Print?
> I use M420 S1 in startcode

Remember that reflashing firmware deletes the mesh in this case.

[Wiki - BL Touch](https://github.com/Jyers/Marlin/wiki/Configuration-Details#bl-touch)
_**This mesh is stored in eeprom and will need to be remade after re-flashing your firmware. M420 S1 is required in your start Gcode to enable the mesh.**_

**Comment by Jyers** [comment, marked as answer]
The mesh active state is not persistent on a power cycle, as long as you have M420 in your start gcode your mesh will be applied.

**Reply by Tuncay-Ayhan** [reply]
> The mesh active state is not persistent on a power cycle, as long as you have M420 in your start gcode your mesh will be applied.

This should be added to the wiki. Now you mention it, I remember someone else asked the same question a few days/weeks ago.
